src/components/NewMovie/NewMovie.tsx

```diff
@@ -13,33 +48,50 @@ export const NewMovie = () => {
       <TextField
         name="title"
         label="Title"
-        value=""
-        onChange={() => {}}
+        value={title}
+        onChange={(newTitle) => {
+          setTitle(newTitle);
+        }}
```

Comment:
Try to type this, should work
```suggestion
        onChange={setTitle}
```